tools/base/tests/test_runner.py

```diff
@@ -0,0 +1,83 @@
+import importlib
```

Comment:
I got confused for a second and thought this might be a test runner, but it's a test for `runner.py`. In Envoy we have the convention of `X_test.cc`, can we do this?

Reply:
this is a test - the convention for pytest/python tests is that you prefix the test module with `test_`